#!/usr/bin/env python3
PACKAGE = 'eagle_mpc_viz'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("continuous_player", bool_t, 0, "Check to reproduce trajectory in continuous mode", False)
gen.add("trajectory_percentage", double_t, 0, "Percentage of the whole of the trajectory", 0., 0., 100.)

exit(gen.generate(PACKAGE, "Parameters for multicotper mpc", "Visualization"))
